2014-10-25 13:40:06,862 - [INFO] - from play in pool-35-thread-5 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2014-10-25 14:13:03,707 - [ERROR] - from application in New I/O worker #1 


! @6k20i7ica - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[value itemTitle is not a member of models.Foods.FoodItem]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[na:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[na:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[na:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[na:na]
	at com.typesafe.trace.FutureRunnable.run(FutureTrace.scala:119) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[na:na]

2014-10-25 14:13:04,116 - [WARN] - from play in New I/O worker #1 
No application found at invoker init

2014-10-25 14:13:39,311 - [ERROR] - from application in New I/O worker #1 


! @6k20i9j6j - Internal server error, for (GET) [/] ->

play.PlayExceptions$UnexpectedException: Unexpected exception[The compilation failed without reporting any problem!]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$17.apply(PlayReloader.scala:306) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$17.apply(PlayReloader.scala:306) ~[na:na]
	at scala.Option.getOrElse(Option.scala:120) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:305) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[na:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[na:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[na:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[na:na]
	at com.typesafe.trace.FutureRunnable.run(FutureTrace.scala:119) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[na:na]
sbt.compiler.CompileFailed: null
	at sbt.compiler.JavaCompiler$JavaTool0.compile(JavaCompiler.scala:77) ~[na:na]
	at sbt.compiler.JavaTool$class.apply(JavaCompiler.scala:35) ~[na:na]
	at sbt.compiler.JavaCompiler$JavaTool0.apply(JavaCompiler.scala:63) ~[na:na]
	at sbt.compiler.JavaCompiler$class.compile(JavaCompiler.scala:21) ~[na:na]
	at sbt.compiler.JavaCompiler$JavaTool0.compile(JavaCompiler.scala:63) ~[na:na]
	at sbt.compiler.AggressiveCompile$$anonfun$3$$anonfun$compileJava$1$1.apply$mcV$sp(AggressiveCompile.scala:126) ~[na:na]
	at sbt.compiler.AggressiveCompile$$anonfun$3$$anonfun$compileJava$1$1.apply(AggressiveCompile.scala:126) ~[na:na]
	at sbt.compiler.AggressiveCompile$$anonfun$3$$anonfun$compileJava$1$1.apply(AggressiveCompile.scala:126) ~[na:na]
	at sbt.compiler.AggressiveCompile.sbt$compiler$AggressiveCompile$$timed(AggressiveCompile.scala:159) ~[na:na]
	at sbt.compiler.AggressiveCompile$$anonfun$3.compileJava$1(AggressiveCompile.scala:125) ~[na:na]
	at sbt.compiler.AggressiveCompile$$anonfun$3.apply(AggressiveCompile.scala:142) ~[na:na]
	at sbt.compiler.AggressiveCompile$$anonfun$3.apply(AggressiveCompile.scala:86) ~[na:na]
	at sbt.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:38) ~[na:na]
	at sbt.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:36) ~[na:na]
	at sbt.inc.Incremental$.cycle(Incremental.scala:73) ~[na:na]
	at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:33) ~[na:na]
	at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:32) ~[na:na]
	at sbt.inc.Incremental$.manageClassfiles(Incremental.scala:41) ~[na:na]
	at sbt.inc.Incremental$.compile(Incremental.scala:32) ~[na:na]
	at sbt.inc.IncrementalCompile$.apply(Compile.scala:26) ~[na:na]
	at sbt.compiler.AggressiveCompile.compile2(AggressiveCompile.scala:150) ~[na:na]
	at sbt.compiler.AggressiveCompile.compile1(AggressiveCompile.scala:70) ~[na:na]
	at sbt.compiler.AggressiveCompile.apply(AggressiveCompile.scala:45) ~[na:na]
	at sbt.Compiler$.apply(Compiler.scala:70) ~[na:na]
	at sbt.Defaults$.sbt$Defaults$$compileTaskImpl(Defaults.scala:722) ~[na:na]
	at sbt.Defaults$$anonfun$compileTask$1.apply(Defaults.scala:716) ~[na:na]
	at sbt.Defaults$$anonfun$compileTask$1.apply(Defaults.scala:716) ~[na:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[na:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_20]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_20]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_20]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_20]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_20]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_20]

2014-10-25 14:14:24,922 - [INFO] - from play in play-internal-execution-context-1 
datasource [jdbc:mysql://localhost/restaurantdb] bound to JNDI as DefaultDS

2014-10-25 14:14:25,336 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/restaurantdb

2014-10-25 14:14:26,190 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-10-25 14:16:55,588 - [INFO] - from play in play-internal-execution-context-1 
datasource [jdbc:mysql://localhost/restaurantdb] bound to JNDI as DefaultDS

2014-10-25 14:16:55,633 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/restaurantdb

2014-10-25 14:16:55,734 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-10-25 14:17:44,316 - [ERROR] - from application in New I/O worker #2 


! @6k20inhi5 - Internal server error, for (GET) [/food/6] ->

play.PlayExceptions$CompilationException: Compilation error[value franchise is not a member of models.Foods.FoodItem]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[na:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[na:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[na:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[na:na]
	at com.typesafe.trace.FutureRunnable.run(FutureTrace.scala:119) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[na:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[na:na]

2014-10-25 14:18:08,328 - [INFO] - from play in play-internal-execution-context-3 
datasource [jdbc:mysql://localhost/restaurantdb] bound to JNDI as DefaultDS

2014-10-25 14:18:08,376 - [INFO] - from play in play-internal-execution-context-3 
database [default] connected at jdbc:mysql://localhost/restaurantdb

2014-10-25 14:18:08,462 - [INFO] - from play in play-internal-execution-context-3 
Application started (Dev)

2014-10-25 14:18:39,078 - [INFO] - from play in play-internal-execution-context-3 
datasource [jdbc:mysql://localhost/restaurantdb] bound to JNDI as DefaultDS

2014-10-25 14:18:39,128 - [INFO] - from play in play-internal-execution-context-3 
database [default] connected at jdbc:mysql://localhost/restaurantdb

2014-10-25 14:18:39,213 - [INFO] - from play in play-internal-execution-context-3 
Application started (Dev)

2014-10-25 15:43:25,975 - [INFO] - from play in play-internal-execution-context-4 
datasource [jdbc:mysql://localhost/restaurantdb] bound to JNDI as DefaultDS

2014-10-25 15:43:26,033 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:mysql://localhost/restaurantdb

2014-10-25 15:43:26,147 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2014-10-25 16:11:04,049 - [INFO] - from play in play-internal-execution-context-4 
datasource [jdbc:mysql://localhost/restaurantdb] bound to JNDI as DefaultDS

2014-10-25 16:11:04,092 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:mysql://localhost/restaurantdb

2014-10-25 16:11:04,198 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

